### PR TITLE
Pass client UTC offset to audio and video client for metrics

### PR DESCRIPTION
**Pull request by aman-aguptaaa**
## ℹ️ Description
*provide a summary of the changes and the related issue, relevant motivation and context*
Pass client UTC offset to audio and video client for metrics. This is being done for investigation of audio quality scores with the purpose of improving the audio quality for Amazon chime sdk customer.
### Issue #, if available
None

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
    - [ ] README update
    - [x] CHANGELOG update
    - [ ] guides update
- [x] This change requires a dependency update
    - [x] Amazon Chime SDK Media
    - [ ] Other (update corresponding legal documents)

## 🧪 How Has This Been Tested?
*describe the tests that you ran to verify your changes, any relevant details for your test configuration*
Tested by running ios demo app successfully with the changes being proposed.

### Additional Manual Test
- [x] Pause and resume remote video
- [x] Switch local camera
- [x] Rotate screen back and forth

## 📱 Screenshots, if available
*provide screenshots/video record if there's a UI change in demo app*

#### Integration validation (required before release)

- [x] Unit tests pass
- [ ] Build SDK against simulator with release options
- [x] Build SDK against device with release options
- [ ] Build SDK against simulator with debug options
- [ ] Build SDK against device with debug options
- [ ] Test Demo against simulator with SDK (debug build) in workspace
- [ ] Test Demo against device with SDK (debug build) in workspace
- [ ] Test DemoObjC against simulator with SDK (debug build) in workspace
- [ ] Test DemoObjC against device with SDK (debug build) in workspace
- [ ] Test Demo against simulator with SDK.framework (release build)
- [x] Test Demo against device with SDK.framework (release build)
- [ ] Test DemoObjC against simulator with SDK.framework (release build)
- [x] Test DemoObjC against device with SDK.framework (release build)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: AmazonChimeSDK/AmazonChimeSDK/internal/utils/DeviceUtils.swift [hunk 1]
```diff
@@ -60,9 +61,18 @@ import UIKit
         appInfo.deviceMake = "apple"
         appInfo.clientSource = "amazon-chime-sdk"
         appInfo.chimeSdkVersion = Versioning.sdkVersion()
+        appInfo.clientUtcOffset = getClientUtcOffset()
         return appInfo
     }
 
+    static private func getClientUtcOffset() -> String {
+        let seconds = TimeZone.current.secondsFromGMT()
+        let hours = seconds/3600
```
Comment (linsang21) [inline AmazonChimeSDK/AmazonChimeSDK/internal/utils/DeviceUtils.swift:70]: Same comment as Android: create constants with property names for the numbers

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
+## Unreleased
+
+### Added
+* Pass client UTC offset to audio and video client for metrics.
+
+### Changed
```
Comment (linsang21) [inline CHANGELOG.md:6]: No need to create place holder for empty sections

File: AmazonChimeSDK/AmazonChimeSDK/internal/utils/DeviceUtils.swift [hunk 3]
```diff
@@ -60,9 +61,18 @@ import UIKit
         appInfo.deviceMake = "apple"
         appInfo.clientSource = "amazon-chime-sdk"
         appInfo.chimeSdkVersion = Versioning.sdkVersion()
+        appInfo.clientUtcOffset = getClientUtcOffset()
         return appInfo
     }
 
+    static private func getClientUtcOffset() -> String {
```
Comment (linsang21) [inline AmazonChimeSDK/AmazonChimeSDK/internal/utils/DeviceUtils.swift:68]: Do we except unit tests for this?

Comment (aman-aguptaaa) [reply AmazonChimeSDK/AmazonChimeSDK/internal/utils/DeviceUtils.swift:68]: Actually there were no unit tests for this file. I have moved this function to DateUtils and added unit tests for it.

**Review by georgezy-amzn — APPROVED** [review]

**Review by linsang21 — APPROVED** [review]

**Review by hokyungh — APPROVED** [review]

**Pull request merged**